README.md

```diff
@@ -2,10 +2,9 @@
   <a href="https://runme.io"><img src="https://runme.io/static/button.svg" alt="Runme" width="200"></a>
 </h1>
 
-<p align="center">
-  <a href="https://chat.jexia.com"><img src="https://img.shields.io/badge/chat-on%20discord-7289da.svg?sanitize=true" alt="Discord"></a>
-  <a href="https://standardjs.com/"><img src="https://img.shields.io/badge/code%20style-standard-brightgreen.svg" alt="Standard - JavaScript Style Guide"></a>
-</p>
+![https://chat.jexia.com](https://img.shields.io/badge/chat-on%20discord-7289da.svg?sanitize=true)
+![https://standardjs.com/](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)
+![Release](https://github.com/runme-io/website/workflows/Release/badge.svg?branch=master)
```

Comment:
Either you remove all html tags and make the header all aligned left or keep everything at center:

**Before**
![Screenshot 2020-06-23 at 13 24 40](https://user-images.githubusercontent.com/4020905/85398117-03462e00-b555-11ea-90f3-fc042025eabb.png)

**After**
![Screenshot 2020-06-23 at 13 24 33](https://user-images.githubusercontent.com/4020905/85398156-148f3a80-b555-11ea-969b-fe2c5caf655e.png)

Reply:
fixed